docs/user/whats-new.asciidoc

```diff
@@ -1,12 +1,208 @@
+Highlight an anomaly in time and see a visual indication
+of this time on other panels.
+The reverse is true as well&mdash;hovering in another editor highlights the same point-in-time in *{ml-app}* panels.
+
+[float]
+=== Canvas
+
+[float]
+==== Native heatmap
+Create heatmaps in Canvas and natively use Canvas-only functionality, such as ES SQL,
+to fetch data. Fully configure your heatmap with flexible legend, axis,
+and visual controls.
+
+[float]
+==== New filter panel
+Use the new filter panel on any object to see which objects are filtering it.
+This helps you author your workpad with the right behavior.
+
+[role="screenshot"]
+image::images/highlights-canvas-filter.png[Filter panel in Canvas]
+
+[float]
+=== Visualizations
+
+[float]
+==== Combine fields with drag and drop
+
+Combine multiple fields with drag and drop to see top combinations in Lens.
+Use this feature to determine which combinations contribute to your metrics’ performance.
+
+[role="screenshot"]
+image::images/highlights-lens-combine.gif[Drag to combin in Lens]
+
+[float]
+==== Top values by rarity
+
+When looking for root cause, it is often useful to find rare values in your data.
+Do this in Lens by seeing top values by rarity.
+This takes advantage of the {es} rare terms aggregation and makes it easy to ask
+“What’s not normal here?”
+
+[role="screenshot"]
+image::images/highlights-lens-rarity.png[Rank by rarity in top values in Lens, width=50%]
+
+[float]
+==== Horizontal and vertical gauges
+
+The gauge visualization adds extra context to your metrics.
+Show minimum and maximum values dynamically by using a quick function or custom formula,
+or set a static value for known metric ranges. Include a goal value to see
+if a metric is above or below the goal. Set color bands to show ranges visually&mdash;imilar
+to reference lines in bar, line, and area visualizations. This new gauge visualization is
+in technical preview in 8.1.
+
+[role="screenshot"]
+image::images/highlights-lens-guage.png[Example horizontal gauge with target value in Lens, width=50%]
+
+[float]
+==== Mosaic visualization
+
+The mosaic visualization, also known as marimekko or mekko, allows for side-by-side
+comparison of two dimensions against a single metric. Treemaps and two-layered
+pies do this as well, but the mosaic can be more effective for visual comparisons
+by aligning the colors for the term side by side. Try this on low cardinality
+dimensions to see its strengths. This new mosaic visualization is in technical preview in 8.1.
+
+[role="screenshot"]
+image::images/highlights-lens-mosaic.png[Example mosaic visualization in Lens]
+
+[float]
+==== Waffle visualization
+
+The waffle visualization is a proportion visualization that makes
+more efficient use of space than pie charts.
+Waffles are designed for square spaces, which aligns well with how
+most dashboards are authored. Waffles are stronger for low cardinality breakdowns,
```

Comment:
```suggestion
most dashboards are created. Since waffles display the data in 1% blocks, they are best for low cardinality breakdowns, and are not recommended for small partitions. 
```